v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerIT/mysql-session.json

```diff
@@ -415,6 +415,129 @@
       "ForeignKeys": null,
       "Indexes": null,
       "Id": "t2"
+    },
```

Comment:
As discussed, move these changes to DataStreamToSpannerSessionIT and add a test case method there (droppedTableTest).